### ISSUE TITLE
Create version compatible with akka-http 2.0-M1

### ISSUE THREAD

**Issue by francisdb** [hseeberger/akka-http-json#32]
Current version is not compatible (as expected)
```
root[ERROR] java.lang.NoSuchMethodError: akka.http.scaladsl.unmarshalling.Unmarshaller$.byteStringUnmarshaller(Lakka/stream/Materializer;)Lakka/http/scaladsl/unmarshalling/Unmarshaller;
root[ERROR] 	at de.heikoseeberger.akkahttpplayjson.PlayJsonSupport$class.playJsValueUnmarshaller(PlayJsonSupport.scala:44)
```

**Comment by hseeberger** [comment]
Released 1.2.0.

**Comment by francisdb** [comment]
that is fast :+1: thanks
